src/main/java/org/jenkinsci/test/acceptance/plugins/git/GitScm.java

```diff
@@ -29,13 +29,21 @@
 import org.openqa.selenium.support.ui.Select;
 
 import javax.swing.text.html.HTMLDocument;
+import java.lang.reflect.InvocationTargetException;
 import java.util.ArrayList;
 
 @Describable("Git")
 public class GitScm extends Scm {
+
     private final Control branch = control("branches/name");
     private final Control url = control("userRemoteConfigs/url");
     private final Control tool = control("gitTool");
+    private final Control repositoryBrowser = this.control(new String[]{"/"});
```

Comment:
`control` accepts variadic argument list so there is no need to wrap arguemnts to array - especially when there only a single element. Use `control("foo")` or `control("foo", "bar")`

Reply:
Changed